src/utils/paths.js

```diff
@@ -53,27 +60,44 @@ function pushPathCondition(condition: Value) {
     pushPathCondition(left);
     pushPathCondition(right);
   } else {
+    if (condition.kind === "!=" || condition.kind === "==") {
```

Comment:
The increasing asymmetry of `pushPathCondition` vs `pushInversePathCondition` worries me.

Reply:
Good point. There is not reason why pushInversePathCondition should not also handle != and ==.